Cargo.toml

```diff
@@ -27,7 +27,7 @@ clap = { version = "2.33", optional = true }
 regex = {version = "1", optional = true }
 
 [features]
-default = []
+default = ["repl", "electrum"]
```

Comment:
The reason we didn't add any default features is because the `wasm` playground app doesn't need `repl` and can't use `electrum` .. but maybe this is OK if we just build `bitcoindevkit.org/playground` with `default-features = false` and `features=["esplora"]`.  @afilini does that sound ok?

Reply:
Yes, exactly. As long as there's a way to disable them it should be fine